doc/articles/get-started-rider.md

```diff
@@ -36,12 +45,15 @@ uid: Uno.GetStarted.Rider
 
 ## Install the Uno Platform plugin
 
+<!-- TODO: This section needs to be reviewed to determine if we should reference the JetBrains Marketplace instead of directly pointing to the Configure menu in Rider. It appears that the channel options (Stable | Preview) are not available at this level based on my quick first review. -->
```

Comment:
I will need to sync with Daniil on this part to make sure we provide the correct details based on the channels to install the proper plugin version based on the Rider version used.

Reply:
@agneszitte I think we don't need to mention EAP in the docs, Rider has released the RC for 2024.2 
![image](https://github.com/user-attachments/assets/87908890-cc4a-40d2-aaee-ad92cac5b52a)
so generally we may say that users need to target 2024.2 or EAP 8+
https://blog.jetbrains.com/dotnet/2024/08/07/rider-2024-2-rc/

Regarding channels - since we didn't publish anything to the dev yet it isn't visible, but fo other plugins it will usually look like that https://plugins.jetbrains.com/plugin/14839-avaloniarider/versions/dev
![image](https://github.com/user-attachments/assets/9f444225-1cf8-4e03-ad61-0e5b66f306ee)

Reply:
I will make the last adjustments based on that and on our sync this morning regarding the details for the channels

Reply:
@agneszitte We should always use an `aka` link for that kind of external link.

Reply:
I am done with all the adjustments now